microsoft/testsuites/kselftest/kselftest-suite.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Dict
+
+from lisa import Environment, Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.testsuite import TestResult
+from microsoft.testsuites.kselftest.kselftest import Kselftest
+
+
+@TestSuiteMetadata(
+    area="kselftest",
+    category="community",
+    description="""
+    This test suite is used to run kselftests.
+    """,
+)
+class KselftestTestsuite(TestSuite):
+    # kselftests take under an hour to complete, timeout below is in seconds
+    _TIME_OUT = 3600
+
+    @TestCaseMetadata(
+        description="""
+        This test case will run linux kernel self tests.
+        Steps:
+        1. Provide a tarball of kselftest binaries kselftests.tar.xz in a .yml file.
```

Comment:
Can you please update the new logic here?

Reply:
Updated test description to include cloning and building kernel.